SoftwareDesign.md

```diff
@@ -0,0 +1,3 @@
+My favorite design pattern I learned this week was the MVC mode. Breaking the model, view, and controller up allows you to focus specifically on the task at hand, and let them do their jobs well. Although Django uses more of a model, view, template(which is also like a controller), and it is a little tricky for myself to get wrap my head around, I still see the advantages and plan on improving my skills int his language and framework. 
```

Comment:
Nice! MVC is sort of the classic architecture for web apps, It's getting to a point where it's in a weird state right now, as a lot of people are moving into a sort of unidirectional architecture with things like Flow and Redux, but MVC is still probably one of the best ways to handle things on the server side with frameworks like Rails and Django.